readme.markdown

```diff
@@ -90,3 +90,8 @@ Example:
 To develop run `npm start`.
 You can test your changes by opening `localhost:4567` in a browser.
 When you are happy with your changes you can build to dist with `npm run build`.
+
+
+## License
```

Comment:
What do you think about following this?:

```markdown
## License

[MIT](LICENSE) &copy; Purpose Industries
```

I'd just like to show the licence directly in the readme.